client/signup/steps/jpo-contact-form/index.jsx

```diff
@@ -49,21 +49,21 @@ const JPOContactFormStep = React.createClass( {
 	renderStepContent() {
 		return (
 			<div className="jpo__contact-form-wrapper">
-				<a className="jpo-homepage__select-news" href="#" onClick={ this.onSelect }>
-					<Card>
-						<ContactUsGraphic />
-						<Button onClick={ this.onSelect }>
-							{
-								translate( 'Add a contact form' )
-							}
-						</Button>
-						<div className="jpo__contact-form-description">
-							{
-								translate( 'Not sure? You can skip this step and add a contact form later.' )
-							}
+				<Card className="jpo-contact-form__choice">
+					<a className="jpo-contact-form__select-news jpo-contact-form__choice-link" href="#" onClick={ this.onSelect }>
+						<div clasName="jpo-contact-form__image">
+							<ContactUsGraphic />
 						</div>
-					</Card>
-				</a>
+						<div className="jpo-contact-form__choice-copy">
+							<Button className="jpo-contact-form__cta" onClick={ this.onSelect }>
+								{ translate( 'Add a contact form' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **46** times:
`translate( 'Add Contact Form' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).